### ISSUE TITLE
Working on Expanding PCBA Dataset

### ISSUE THREAD

**Issue by LRParser** [deepchem/deepchem#725]
Just noting that I'm working on DeepChem for a Udacity Machine Learning Engineer Nanodegree Capstone Project. My plan is to try to expand the PCBA to include more assays from 2016 and 2017 in the training set, as well as in a separate step to perform hyperparameter optimizations around learning rate and network layers, to see if the performance of the base index-split multitask DNN can be improved with regards to predications against the original index-based test set. The planned work described in more detail here:

https://docs.google.com/document/d/1NclOAuBTAbMAfZa6setZYOlz-3Z5cF-A4JAt8W4-7AE/edit?usp=sharing

Any suggestions you may have are much appreciated. My goal is more to do some useful work and get started on understanding the codebase vs to do something ground-breaking around the graph convolutional code (which I understand has better scores).

**Comment by lilleswing** [comment]
For your hyperparameter optimization piece it seems like @patrickhop and @miaecle have has luck with using https://github.com/hawk31/pyGPGO. See here https://github.com/deepchem/deepchem/issues/720.

When adding the new Assays to PCBA can you add a flag in (pcba_datasets.py)[https://github.com/deepchem/deepchem/blob/master/deepchem/molnet/load_function/pcba_datasets.py] which allows for using the older dataset for MoleculeNet reproducability?  Something like

``` python
def load_pcba(featurizer='ECFP', split='random', reload=True, version='2017'):
  """Load PCBA datasets."""
```

Sounds like an interesting project!  Wish you the best!

**Comment by rbharath** [comment]
@LRParser Glad to hear about this project! I'm looking forward to seeing what you build.

Seconding @lilleswing's comment about the flag.

**Comment by LRParser** [comment]
Thanks for feedback - will do!

**Comment by rbharath** [comment]
@LRParser Thanks for the PRs you've submitted as part of this project! I think they've added a lot of value to the project and will be a good reference for people building on your work.

Going to go ahead and close this issue. Please re-open if there are additional issues to discuss on the topic.
